apple/Text/RNSVGTopAlignedLabel.ios.m

```diff
@@ -0,0 +1,21 @@
+#if !TARGET_OS_OSX
```

Comment:
Just get rid of the if def and only build into iOS targets?

Reply:
We already exclude *.ios.m files on macOS as per the podspec file. I just threw the ifdef in as a "just in case" sort of thing, but yeah, we technically don't need it.